app/models/form_handler.rb

```diff
@@ -7,29 +7,50 @@ def initialize
   end
 
   def get_form(form)
-    @forms[form]
+    @forms[form]["form"] if @forms[form].present?
   end
 
-  def current_form
-    forms[forms.keys.max_by(&:to_i)]
+  def current_lettings_form
+    forms["current_lettings"]["form"]
+  end
+
+  def current_sales_form
+    forms["current_sales"]["form"]
   end
 
   def sales_forms
     sales_sections = [] # Add section classes here e.g. Form::Sales::Property::Sections::PropertyInformation
-    { "2022_2023_sales" => Form.new(nil, "2022_2023_sales", sales_sections, "sales") }
+    current_form = Form.new(nil, current_collection_start_year, sales_sections, "sales")
+    previous_form = Form.new(nil, current_collection_start_year - 1, sales_sections, "sales")
+    { "current_sales" => { "form" => current_form, "type" => "sales", "start_year" => current_form.start_date.year },
```

Comment:
```suggestion
    { "current_sales" => { "form" => current_form, "type" => "sales", "start_year" => current_form.start_date.year },
```

I wonder if we need `type` and `start_year` here as keys if both can be inferred from the form itself. It might simplify this to:

```suggestion
    {
      "current_sales" => Form.new(nil, current_collection_start_year, sales_sections, "sales"), 
      "previous_sales" => Form.new(nil, current_collection_start_year, sales_sections, "sales"), 
    }
```

What do you think?

Reply:
I mean, it does make loads of sense, cause we can get the year and the type from the form itself. 
Let me give this a go

Reply:
Oh lol, sorry, didn't see your actual comment, but I agree 😅